frontend/src/Status.jsx

```diff
@@ -1,32 +1,62 @@
 import React, { Component } from 'react';
 import { Jumbotron, Button, Panel } from 'react-bootstrap';
 import StatusCheck from './components/status/StatusCheck.jsx';
+import SystemCheck from './components/status/SystemCheck.jsx';
+import detectBrowser from './helpers/DetectBrowser';
+import getIeVersion from './helpers/GetIeVersion';
+import getScreenResolution from './helpers/GetScreenResolution';
 
 const styles = {
   container: {
     width: '50%',
     margin: '0 auto',
   },
+  panel: {
+    padding: '0px',
+  },
+  table: {
+    width: '100%',
+  },
+  td: {
+    border: '0.5px solid #dddddd',
+  },
 };
 
 const headers = new Headers({
   Accept: 'application/json',
   'Content-Type': 'application/json',
 });
 
+// Valid browser other than IE
+const validBrowser = ['chrome', 'firefox'];
```

Comment:
Constants should be all caps: https://github.com/airbnb/javascript#naming-conventions